scrapper.py

```diff
@@ -44,20 +75,94 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(article_url, article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        article_title = article_bs.find('h1').text
+        self.article.title = article_title.strip()
+
+        self.article.author = 'NOT FOUND'
```

Comment:
so you do not have author for sure?

Reply:
Yes, I didn't find any authors in the articles on my resource